### PR TITLE
Add noInputContainerPlacementStrategy.

### PR DESCRIPTION
**Pull request by evanchaoli**
Related to https://github.com/concourse/concourse/pull/8183

# Contributor Checklist

- [x] Variables are documented in the `README.md`
- [x] Which branch are you merging into?
    - `master` is for changes related to the current release of the `concourse/concourse:latest` image and should be good to publish immediately
    - `dev` is for changes related to the next release of Concourse (aka unpublished code on `master` in [concourse/concourse](https://github.com/concourse/concourse))

# Reviewer Checklist
> This section is intended for the core maintainers only, to track review progress. Please do not
> fill out this section.
- [ ] Code reviewed
- [ ] Topgun tests run
- [ ] Back-port if needed
- [ ] Is the correct branch targeted? (`master` or `dev`)

### CODE REVIEW

File: values.yaml [hunk 1]
```diff
@@ -329,6 +329,19 @@ concourse:
     ## Possible values: volume-locality | fewest-build-containers | limit-active-tasks | limit-active-containers | limit-active-volumes
     containerPlacementStrategies: []
 
+    ## A second method by which a worker is selected during container placement
+    ## that is only used for get and nested check steps.
+    ## Possible values: volume-locality | random | fewest-build-containers | limit-active-tasks | limit-active-containers | limit-active-volumes
+    noInputContainerPlacementStrategy:
```
Comment (xtremerui) [inline values.yaml:335]: Is this one necessary? If noInputContainerPlacementStrategies contains only one strategy, isn't it the same as this one?

Comment (evanchaoli) [reply values.yaml:335]: @xtremerui  I thought here just lists all options with default values. Is it true?

Comment (xtremerui) [reply values.yaml:335]: hmm, i just notice `containerPlacementStrategy` and `containerPlacementStrategy`. We are good here.

**Review by clarafu — APPROVED** [review]
Thanks!!

**Pull request merged**